assembl/tests/fixtures/graphql.py

```diff
@@ -4,12 +4,29 @@
 
 @pytest.fixture(scope="function")
 def graphql_request(request, test_adminuser_webrequest, discussion):
+    """ A graphql request fixture with an ADMIN user authenticated """
     req = test_adminuser_webrequest
     req.matchdict = {"discussion_id": discussion.id}
     req.method = 'POST'
     return req
 
 
+@pytest.fixture(scope="function")
+def graphql_unauthenticated_request(request, test_webrequest, discussion_with_default_data):
+    req = test_webrequest
+    req.matchdict = {"discussion_id": discussion_with_default_data.id}
```

Comment:
you really need `discussion_with_default_data` here? from what I understood, it creates more stuff and thus takes more time. If it is just to test unauthenticated, `discussion` may suffice, isn't it?

Reply:
In this particular case yes, `discussion` suffice. `discussion_with_default_data` is needed in the case where you test a permission with a non admin account.

Reply:
I think I do. It's hat fixed the failing tests

Reply:
https://travis-ci.org/assembl/assembl/jobs/327716230

Reply:
You're right, the unauthencited user Everyone is not a special case, only the admin user having R_SYSADMIN is, see auth/util.py:get_permissions. For a user not having R_SYSADMIN, you need all DiscussionPermission created by `discussion_with_default_data`.

Reply:
ok, good to know